src/Redux/Store/Expeditions/Expeditions/actions.ts

```diff
@@ -0,0 +1,62 @@
+import { createAction } from '@martin_hotell/rex-tils'
+
+import * as types from 'types'
+
+import {
+  ActionTypes,
+  State,
+  BaseConfig,
+  LossConfig,
+  RollBattleConfig,
+  WinConfig,
+} from './types'
```

Comment:
These are already imported by line 3. Can we delete line 3 though?

Reply:
Nevermind, I just saw the tiny difference between them 😄

Reply:
@jannydiamond typescript would complain if we had duplicate import entry points ;)
So its pretty hard to manually fuck up here :)